package.json

```diff
@@ -23,19 +23,20 @@
         "commit": "! git diff --cached --exit-code && lint-staged && git-cz",
         "install:clean": "lerna clean --yes && yarn install",
         "lint": "eslint . --ext ts,js,md",
-        "lint:all": "tsc --noEmit && yarn lint && yarn lint:lockfile && yarn lint:deps && yarn lint:depgraph",
+        "lint:all": "tsc --noEmit && yarn lint && yarn lint:lockfile && yarn lint:deps && yarn lint:depgraph && yarn lint:version",
         "lint:deps": "lerna exec depcheck",
         "lint:depgraph": "yarn pkg:depgraph && git diff --quiet docs/sa11y_dependency_graph.svg || echo 'Dependency graph needs to be updated!'",
         "lint:fix": "yarn lint --fix",
         "lint:lockfile": "lockfile-lint --path yarn.lock --allowed-hosts registry.yarnpkg.com --validate-https",
         "lint:staged": "lint-staged",
+        "lint:version": "vertioner && yarn lerna exec --since master vertioner",
```

Comment:
It is a small util to check if the `version` field in `package.json` file in current branch is different than master. I am using it to ensure the versions for packages that are changed are bumped without fail.

Reply:
So the expectation is every PR must bump the version of all packages?

Reply:
Not really -  only the changed packages (`yarn lerna exec --since master ..` takes care of that) and the monorepo root version.

Reply:
Ok interesting. I see a couple potential issues with this flow:

1. There can be version conflicts between PRs. Imagine there are several PRs open at the same time and they all bump the version to the next patch version. As soon as one gets merged, all other PRs must go in and bump their version again. This potentially repeats for every open PR when a different PR is merged. 

2. There may be versions in github without corresponding NPM releases. Unless we release every commit to master to NPM, there may be several version bumps before a new version in NPM. Not necessarily a bad thing but can be confusing IMO.

3. We put the burden of versioning on the PR author. When I contribute to other projects I just focus on the code fix rather than worrying about whether something should be a patch vs. minor release. 

Just my opinion but I think it makes more sense to leave the versions untouched in PRs and then do a release from `master` or a release branch and we as project maintainers decide when that release happens and what version to bump to.

Reply:
Thanks @trevor-bliss - Those are very valid concerns 👍 
I have created a issue to track this and continue discussion https://github.com/salesforce/sa11y/issues/40

Reply:
@trevor-bliss I have removed the version check on mono-repo root due to [a potential issue of it failing after merging into master](https://github.com/hectotruj/vertioner/issues/5)

I was also able to remove force setting the master branch in CI branch since `lerna exec` seems to work fine without it. 

Can you please give a glance and approve. 
I will release to npm and work on improving versioning as part of #40